docs/source/installation.rst

```diff
@@ -9,99 +9,180 @@
     Use the version switcher in the top-right corner of the page to download/install other versions.
+
+
+    Alternative installation methods
+    ================================
+    Although *not currently recommended*,
+    advanced users may wish to use one of the :ref:`alternative installation methods <alternative_install>`.
+
+
+    #############
+    Configuration
+    #############
+
+    MatFlow uses a config file to control details of how it executes workflows.
+    A :ref:`default config file <default_config>` will be created the first time you submit a workflow.
+    This will work without modification on a personal machine,
+    however if you are using MatFlow on HPC you will likely need to make some
+    modifications to describe the job scheduler, and settings for multiple cores,
+    and to point to your MatFlow environments file.
+
+    `Some examples <https://github.com/hpcflow/matflow-configs>`_ are given
+    for the University of Manchester's CSF.
+
+    The path to your config file can be found using ``matflow manage get-config-path``,
+    or to open the config file directly, use ``matflow open config``.
+
+    #############
+    Environments
+    #############
+
+    Matflow has the concept of environments, similar to python virtual environments.
+    These are required so that tasks can run using the specific software they require.
+    Your MatFlow environments must be defined in your environments (YAML) file before MatFlow
+    can run workflows, and this environment file must be pointed to in the config file
+    via the ``environment_sources`` key.
+    Once this has been done,
+    your environment file can be be opened using ``matflow open env-source``.
+
+    You may wish to modify this template environments file for your own computer,
+    in particular the ``setup`` sections for each environment.
+
+    .. code-block:: yaml
+
+	- name: damask_parse_env
+	  setup: |
+	    source /full/path/to/.venv/bin/activate
+	  executables:
+	    - label: python_script
+	      instances:
+		- command: python <<script_name>> <<args>>
+		  num_cores:
+		    start: 1
+		    stop: 32
+		  parallel_mode: null
+
+	- name: formable_env
+	  setup: |
+	    source /full/path/to/.venv/bin/activate
+	  executables:
+	    - label: python_script
+	      instances:
+		- command: python <<script_name>> <<args>>
+		  num_cores:
+		    start: 1
+		    stop: 32
+		  parallel_mode: null
+
+	- name: defdap_env
+	  setup: |
+	    source /full/path/to/.venv/bin/activate
+	  executables:
+	    - label: python_script
+	      instances:
+		- command: python <<script_name>> <<args>>
+		  num_cores:
+		    start: 1
+		    stop: 32
+		  parallel_mode: null
+
+	- name: damask_env
+	  setup: |
+	    module load mpi/intel-18.0/openmpi/4.1.0
+	    IMG_PATH=/mnt/eps01-rds/jf01-home01/shared/software/DAMASK-docker-images/damask-grid_3.0.0-alpha7.sif
```

Comment:
Can you change this to a fake path, similar to others?

Reply:
Done